xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -498,6 +498,9 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
       m_mime = "video/hevc";
       m_formatname = "amc-hevc";
 
+      bool convertDovi = CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
```

Comment:
```suggestion
      const auto settings = CServiceBroker::GetSettingsComponent()->GetSettings();
      const bool convertDovi =
          (settings) ? settings->GetBool(CSettings::SETTING_VIDEOPLAYER_CONVERTDOVI) : false;
```

We can take the opportunity to use `const bool` and handle the rare case where settings may be null

Reply:
Done.